app/models/concerns/messageable.rb

```diff
@@ -0,0 +1,81 @@
+module Messageable
+  def display_after_initialize_message
```

Comment:
This methods in module are not model methods, these are used as callback methods so put it under **private**